src/otx/algo/classification/efficientnet.py

```diff
@@ -432,16 +433,24 @@ def _create_model(self) -> nn.Module:
         model.init_weights()
         return model
 
+    def label_smoothing_loss(self, output, target, num_classes, smoothing=0.1):
```

Comment:
It looks like torch supports label smoothing out of the box: https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html

Reply:
If it's the same, it would be good to change this, but if it's different, I would move this loss function to the `algo/classification/loss` folder instead of this file.